app/assets/stylesheets/finder_frontend.scss

```diff
@@ -66,6 +61,18 @@
   }
 }
 
+.document--top {
+  border: 1px solid govuk-colour('grey-2');
+  padding: govuk-spacing(6);
+  margin-bottom: govuk-spacing(3);
+
+  .document__link {
```

Comment:
Does this have to be nested inside `.document--top`?  One of the benefits of using BEM is not having to be so specific have have nested classes.